pkg/server/endpoints/entryfetcher.go

```diff
@@ -72,3 +77,18 @@ func (a *AuthorizedEntryFetcherWithFullCache) RunRebuildCacheTask(ctx context.Co
 		}
 	}
 }
+
+func (a *AuthorizedEntryFetcherWithFullCache) PruneEventsTask(ctx context.Context) error {
+	for {
+		select {
+		case <-ctx.Done():
+			a.log.Debug("Stopping event pruner")
+			return nil
+		case <-a.clk.After(a.eventsPruneInterval):
```

Comment:
so, every 12hs it will try to remove all entries that are older than 12hs, 
if we have `e1`, `e2`, `e3` and `e1` was created 13hs ago, and `e2` 11hs ago, 
if we run prune, `e1` will be removed now, but `e2` will be removed in another 12hs (it will have 23hs) 

I'm not sure if we must have 2 intervals, 
first to know how ofter we try to remove (it can be every 1m or 10m or 1h) and 
another one to know the older entries we support.

Reply:
It has been a while, but I believe the cache_reload_interval is now used to see how often to check the cache, and the eventsPruneInterval is the time from "now" into the past that sets the pruning limit.

Reply:
We can do that way and prune every time we reload the cache. Though there would be more delete operations.

Can also have the pruner run at a faster rate than it prunes. So runs every 6 hours and prunes everything over 12 hours old.

Reply:
Changed the name of the configurable to `PruneEventsOlderThan` and have the pruner task trigger at half of that.